### PR TITLE
net-misc/unison: remove old

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: net-misc/unison/unison-2.51.4_rc2.ebuild [hunk 1]
```diff
@@ -25,6 +25,8 @@ RDEPEND="gtk? ( dev-ml/lablgtk:2=
 	|| ( net-misc/x11-ssh-askpass net-misc/ssh-askpass-fullscreen ) )
 	>=app-eselect/eselect-unison-0.4"
 
+QA_FLAGS_IGNORED="/usr/bin/unison{,-fsmonitor}-2.51"
+
```
Comment (juippis) [inline net-misc/unison/unison-2.51.4_rc2.ebuild:29]:
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/unison-fsmonitor-2.51
 * /usr/bin/unison-2.51
 * QA Notice: Files built without respecting LDFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/unison-2.51
 * /usr/bin/unison-fsmonitor-2.51
```
This doesn't seem to work though :I 

I can merge the cleanups only if you wish, but this'd seem pointless. Just let me know.

**Review by juippis — APPROVED** [review]

File: net-misc/unison/unison-2.51.4_rc2.ebuild (hunk 1 above)
Comment (vaukai) [reply net-misc/unison/unison-2.51.4_rc2.ebuild:29]: Please disregard this PR ...

Comment (juippis) [reply net-misc/unison/unison-2.51.4_rc2.ebuild:29]: Commit, or PR? I see 3 commits in here, where 2 could be merged (the cleanups)

Comment (vaukai) [reply net-misc/unison/unison-2.51.4_rc2.ebuild:29]: If you like those cleanups please use them.  I won't do further contributions to this package.

Comment (juippis) [reply net-misc/unison/unison-2.51.4_rc2.ebuild:29]: Will do, thanks!

**Pull request closed without merging**